package-lock.json

```diff
@@ -7143,9 +7143,9 @@
       "integrity": "sha512-gQutuDHPKNxUEcQ4pypZT4Wmrbapus+P9s3bR/SEOLsMbNqNoXigGImITygI5zhb+aA5rzflM6O8YWkmRbGkPA=="
```

Comment:
FYI @m-akinc @jattasNI 
FAST released the changes from https://github.com/microsoft/fast/pull/6667 and https://github.com/microsoft/fast/pull/6879 which this PR will update from `@microsoft/fast-foundation@2.49.5`

After merged we should:
- [ ] Validate the following theme provider bug is fixed in nimble: https://github.com/ni/nimble/issues/1661
- [ ] Remove workarounds added for that theme provider bug in SystemLinkShared (@mollykreis do you know what needs to be updated?)
- [ ] Remove the following workaround from nimble: https://github.com/ni/nimble/blob/main/packages/nimble-components/src/anchor-menu-item/index.ts#L107-L122

Reply:
>Remove workarounds added for that theme provider bug in SystemLinkShared (@mollykreis do you know what needs to be updated?)

The workaround that needs to be removed is in the Security UI. It got added [here](https://dev.azure.com/ni/DevCentral/_git/Skyline/commit/d1c7257a67b593403506d14e7ced1f3174398e78?refName=refs/heads/master&path=/Web/Workspaces/Security/src/app/app.component.ts).

Reply:
I will do all these.

Reply:
@vikisekarNI your listed as CODEOWNER for rich-text components. FYI the latest patch updates of tiptap dependencies in this PR are causing rich-text test failures and need to be addressed to unblock this work.

Reply:
Well, we could split out the PR but it would be nice if they were addressed and the tests may be catching behaviours that apps may find if they update their dependencies.

Reply:
@m-akinc not sure what your bandwidth looks like but lets not block the fast-foundation updates and validation on rich text issues. Bumping the foundation version can be done in a separate PR if you have the time.

Reply:
Captured the remaining checklist as acceptance criteria in #1661